modules/meeting/app/models/meeting_agenda_item.rb

```diff
@@ -45,7 +45,11 @@ class MeetingAgendaItem < ApplicationRecord
 
   scope :with_includes_to_render, -> { includes(:author, :meeting) }
 
-  validates :meeting_id, presence: true
+  # The primer form depends on meeting_id being validated, even though Rails pattern would suggest
```

Comment:
Can we create a follow up ticket to maybe fix this issue? The validation should always be on the association, not on the DB field.

Reply:
Done 👍 https://community.openproject.org/work_packages/53065